src/LazySet.jl

```diff
@@ -33,6 +33,8 @@ Every concrete `LazySet` must define the following functions:
     `Real`
 - `dim(S::LazySet)::Int` -- the ambient dimension of `S`
 
+The subytpes of `LazySet`, including abstract interfaces:
```

Comment:
I am not sure if this is interesting, because it does not contain all interfaces. But okay...

Reply:
Ok, i see. Would be nice to have an option to list all abstract interfaces as well...